.eleventy.js

```diff
@@ -33,6 +33,10 @@ module.exports = function(eleventyConfig) {
   eleventyConfig.addPassthroughCopy("img");
   eleventyConfig.addPassthroughCopy("css");
 
+  if (fs.existsSync("admin")) {
```

Comment:
This section seems very problematic if people aren’t using Stackbit. Is there a better way to opt-in to this? A global copy of `admin` folders seems like a bad idea.